book/modules/module2-exercises.tex

```diff
@@ -90,6 +115,63 @@
 				Since $\vec a-\vec b=\mat{0\\2}$, 
 				$\ell$ is the vertical line containing $\vec b$ (and $\vec a$).
 
+				\item 
+				\begin{tikzpicture}[baseline = (current bounding box.north)]
+					\begin{axis}[
+						anchor=origin,
+						disabledatascaling,
+						xmin=-1,xmax=2,
+						ymin=-1,ymax=1,
+						xtick={-2,...,3},
+						ytick={-2,...,3},
+						x=1cm,y=1cm,
+						grid=both,
+						grid style={line width=.1pt, draw=gray!10},
+						%major grid style={line width=.2pt,draw=gray!50},
+						axis lines=middle,
+						minor tick num=0,
+						enlargelimits={abs=0.5},
+						axis line style={latex-latex},
+						ticklabel style={font=\tiny,fill=white},
+						xlabel style={at={(ticklabel* cs:1)},anchor=north west},
+						ylabel style={at={(ticklabel* cs:1)},anchor=south west}
+					]
+						\fill [mygreen, opacity=.3] (0,0) -- (3,3) -- (3,-3) -- cycle;
+						\draw [black, thick, dashed, ->] (0,0) -- (1,1) node[midway, above left] {$\vec a$};
+						\draw [black, thick, dashed, ->] (0,0) -- (1,-1) node[midway, below left, yshift=2pt] {$\vec b$};
+						\draw [WildStrawberry, thick, -] (1,2) -- (1,-2);
+					\end{axis}
+					\node[WildStrawberry] at (0.75,-1.25) {\small$\ell$};
```

Comment:
If you do `\small` at the start of the picture, you won't need to respecify it each time.

Reply:
Does that mean putting a `font=\small` in the square bracket after `\begin{tikzpicture}`? Also, I noticed that `\small` did not change the size in the graphs, so I plan to remove all the `\small` since the current sizes are okay.

Reply:
It should work to do
```
\begin{tikzpicture}[...]
    \small

\end{tikzpicture}
```

But, now that you mention it, the exercises are already printed in a small font, so there is no reason to reissue the command.